distribution/docs/src/main/resources/_contents/_configuring/configuring-ldap-contents.adoc

```diff
@@ -0,0 +1,163 @@
+
+==== Configuring ${branding} to Use an LDAP Server
```

Comment:
This section describes two optional features and how they can be configured. Putting them in section 3.6 might be a better fit. We should also explain why/when to use them and how to start them using the admin console.